crates/interpreter/src/valid.rs

```diff
@@ -775,6 +820,35 @@ impl<'a, 'm> Expr<'a, 'm> {
         })
     }
 
+    fn br_label_side_table_log_entries(
+        &mut self, l: LabelIdx,
+    ) -> Result<&mut Vec<(&'m [u8], usize)>, Error> {
+        let l = l as usize;
+        let n = self.labels.len();
+        check(l < n)?;
```

Comment:
We don't need to check this because it's already done by `br_label`.

Reply:
Removed the check in `br_label` to satisfy borrow checker.

Reply:
What do you mean? There shouldn't be any borrow checking issue.